### ISSUE TITLE
Several pseudo selectors inside ::slotted() not working

### ISSUE THREAD

**Issue by tobi-or-not-tobi** [chromaui/storybook-addon-pseudo-states#22]
We're using this plugin to preview various states of our (lit based) web components. It works awesome, however we encountered a couple of CSS selectors that are causing the following error:

```
999.3e99130f.iframe.bundle.js:2 DOMException: Failed to execute 'insertRule' on 'CSSStyleSheet': Failed to parse the rule '::slotted(:hover:not(:active)), :host(.pseudo-hover.pseudo-active) ::slotted(:not()) { box-shadow: var(--oryx-elevation-1) var(--oryx-elevation-color); }'.
    at rewriteStyleSheets (https://***/999.3e99130f.iframe.bundle.js:2:2443621)
    at https://***/999.3e99130f.iframe.bundle.js:2:2444564 null
```

The various (valid) selectors that cause this behaviour are:
(adding all of them, but it's a common pattern)

```css
::slotted(:hover:not(:active)) { ... }

::slotted(:is(button, a):active) { .. }

::slotted(:is(button, a):focus-within:focus-visible:not(:active)) { ... }

:host([outline]) ::slotted(:is(button, a):not(:active)) { ... }

:host([outline]) ::slotted(:is(button, a):active) { ... }

:host([outline]) ::slotted(:is(button, a):focus-within:focus-visible:not(:active)) { ... }

:host([type='secondary']) ::slotted(:is(button, a):focus-within:focus-visible:not(:active)) { ... }
```

We could unblock ourselves by changing the selectors, using specific elements. 

```css
 ::slotted(button:hover:not(button:active)),
  ::slotted(a:hover:not(a:active)) { ... }
```
Even the star selector (`*:hover`) or leaving out a specific element (`:hover`) wasn't enough. We not like to changes our selectors and are worried about future component development. 

Any change you can support those selectors?

**Comment by tobi-or-not-tobi** [comment]
We've identified 2 issues when using this addon in combinations with web components:
- `:slotted()` selectors are not supported, #23 has provided a fix for this
- `:is(a, b)` is not supported due to a [split on comma's on the rule](https://github.com/chromaui/storybook-addon-pseudo-states/blob/914dee59c054c8dc34155e478ea69f0131bc5507/src/withPseudoState.js#L99)

Would be great to get this PR (or similar solution) in 🙏

**Comment by tobi-or-not-tobi** [comment]
As the 2nd case (using `is(x, y, z)` was bugging us, we created another PR  (#24) that fixes this.

**Comment by tobi-or-not-tobi** [comment]
Thanks for merging those 2 PR's @ghengeveld. What's the release process for this add-on? We're currently using a customer version in our project, and we'd like to drop this asap.

**Comment by abogdanov-work** [comment]
Additional selector that causes similar issue is pseudo hover combined with some other selector for example:
```
.thumb:hover, .thumb.active {}
```
If `.thumb:hover` is specified by itself it works fine, but we need to use it in case I specified above
